README.md

```diff
@@ -1,5 +1,37 @@
 # md
 
+![summary.png](docs/images/summary.png)
+
+## Installation
+
+```sh
+$ trivy plugin install github.com/dramf/md
+```
+
+## Usage
+### Using CLI
+Formatted report will be saved to the file `trivy-report.md`
 ```sh
-$ trivy image alpine --output plugin=md
+$ trivy image alpine --output plugin=md --format json
 ```
+### Using GitHub Actions
+```sh
+jobs:
+  scan:
+    name: Trivy-scan
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Install plugin
+        run: |
+          trivy plugin install github.com/dramf/md
```

Comment:
Trivy should be installed before this step.

Reply:
Perhaps we can use `skip-trivy-setup` in `trivy-action` (https://github.com/aquasecurity/trivy-action/?tab=readme-ov-file#skipping-setup-when-calling-trivy-action-multiple-times)